src/components/SummaryCard/SummaryCardFooter/__tests__/SummaryCardFooter.spec.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file Summary card footer tests.
+ * @copyright IBM Security 2020
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { SummaryCardFooter } from '../../../..';
+
+describe('SummaryCardFooter', () => {
+  test('should accept a custom class name', () => {
```

Comment:
Will we need a11y tests for these also?

Reply:
All tested together inside `SummaryCard`